wdl/transforms/new-base/src/main/scala/wdl/transforms/base/wdlom2wom/TaskDefinitionElementToWomTaskDefinition.scala

```diff
@@ -57,8 +57,20 @@ object TaskDefinitionElementToWomTaskDefinition {
         }.map(_.toSeq)
       }
 
-      (validRuntimeAttributes, validCommand) mapN { (runtime, command) =>
-        CallableTaskDefinition(a.taskDefinitionElement.name, Function.const(command.validNel), runtime, Map.empty, Map.empty, taskGraph.outputs, taskGraph.inputs, Set.empty, Map.empty)
+      val metaSection : ErrorOr[Map[String, String]] = a.taskDefinitionElement.metaSection match {
+        case None => Map.empty.validNel
+        case Some(MetaSectionElement(meta)) =>
+          val m : Map[String, String] = meta.map{
```

Comment:
We can use [`traverse`](https://typelevel.org/cats/typeclasses/traverse.html#traverse) here to handle constructing the map one element at a time without having to use an exception:
```
meta.traverse {
  case (key, MetaValueElement.MetaValueElementString(value)) =>
    (key -> value).validNel
  case (key, other) =>
    s"non string meta values are not handled currently <${key} -> ${other}>".invalidNel
}
```